backend/LexBoxApi/Controllers/LoginController.cs

```diff
@@ -41,10 +42,14 @@ public class LoginController(
         string returnTo)
     {
         var user = loggedInContext.User;
-        var userUpdatedDate = await userService.GetUserUpdatedDate(user.Id);
-        if (userUpdatedDate != user.UpdatedDate)
+        // A RegisterAccount token means there's no user account yet, so checking UpdatedDate makes no sense
```

Comment:
I still think it would be nice if we could handle a user getting multiple invites.
I think it makes sense to do something like this:
- If it's a `RegisterAccount` audience, check if an account with the email address already exists. If so:
  - Add any projects from this token to that account
  - Give them a new up-to-date token
  - Make the register page smarter (it's sort of a bug that an authenticated user can access the register page right now): if the user is authenticated (and not with `RegisterAccount`) redirect them home

Reply:
I don't think it's realistic that users would be receiving invites to multiple projects at the same time. Our typical user who doesn't have an account yet is a mother-tongue speaker who's being asked to join a language project to contribute his vocabulary/grammar knowledge. Anyone who might be asked to join multiple projects is almost certainly a language consultant, who almost certainly has a Language Depot account already.

I do agree that the registration page could be smarter: now that we have a RegisterAccount audience, only accept a token with that audience and redirect any other tokens home. But I don't think allowing multiple project invitations will be worth the programmer effort involved.

Reply:
created #584

Reply:
Since we're now tracking this idea in #584, I'll resolve this conversation.